data.json

```diff
@@ -166,7 +166,7 @@
     "difficulty": "medium",
     "description": "Submit 3 issues and 1 PR to their <a href='https://github.com/erxes/erxes'>repo</a>, get a t-shirt and stickers.",
     "reference": "https://erxes.io/hubspot-alternative-erxes-swag",
-    "image": "https://erxes.io/static/images/t-shirt.jpeg",
+    "image": "https://i.imgur.com/T1G9zWs.png",
```

Comment:
```suggestion
    "image": "https://i.imgur.com/pE9Ml3v.png",
```

Reply:
Thanks you for the review! I changed the url and amended the change.